src/dashboards/components/AutoRefresh.scss

```diff
@@ -0,0 +1,46 @@
+.refresh-form-container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 10px 50px;
```

Comment:
clockface defines a bunch of standard paddings you should use
I bring it up because it seems like you're using a 5px grid system when our design is built around an 8px grid
using the clockface variables puts it squarely in the "not my problem" bucket